### ISSUE TITLE
Dataset information

### ISSUE THREAD

**Issue by MrYodas** [MolMeDB/MolMeDB#15]
While browsing individual Sets user should obtain short info about selected dataset e.g. membrane and method used, temperature, pH ...

**Comment by KarelBerka** [comment]
so basically, you want to add Comment section?

**Comment by MrYodas** [comment]
Yes ... comment section that would be visible upon selecting individual datasets ... all information about Set is already in system since we they are required while uploading molecules to Set
